adoc/extensions/sycl_khr_queue_flush.adoc

```diff
@@ -0,0 +1,67 @@
+[[sec:khr-queue-flush]]
+= sycl_khr_queue_flush
+
+This extension allows developers to ensure that device code is able to make
+forward progress without the need to call [code]#sycl::queue::wait()#.
+
+[[sec:khr-queue-flush-dependencies]]
+== Dependencies
+
+This extension has no dependencies on other extensions.
+
+[[sec:khr-queue-flush-feature-test]]
+== Feature test macro
+
+An implementation supporting this extension must predefine the macro
+[code]#SYCL_KHR_QUEUE_FLUSH# to one of the values defined in the table below.
+
+[%header,cols="1,5"]
+|===
+|Value
+|Description
+
+|1
+|Initial version of this extension.
+|===
+
+[[sec:khr-queue-flush-funct]]
+== New Queue Command to Force Submission
+
+This extension adds the following function to the [code]#sycl::queue# class.
+
+'''
```

Comment:
I'd prefer that we keep a consistent style between the KHRs.  Our existing sycl_khr_default_context KHR also adds only one member function, yet we still showed the synopsis for the class.

Reply:
Ok, then will copy/paste the sentence. Please let not change it anymore then :D

Reply:
Good point. Fixed with 32e3d1c